app/utils/validations/fields-validations.ts

```diff
@@ -97,6 +97,13 @@ function dateValidation(obj: any, errors): boolean | ValidationErrors {
   const { year, month, day } = obj;
   const date = moment(`${year} ${month} ${day}`, 'YYYY MM DD').isValid() ?
     moment(`${year} ${month} ${day}`, 'YYYY MM DD').format('YYYY MM DD') : 'invalid Date';
+
+  const startDate = new Date();
+  const numOfYears = 18;
+  const cutOfDate = new Date(startDate);
+  cutOfDate.setFullYear(cutOfDate.getFullYear() - numOfYears);
+  cutOfDate.setDate(cutOfDate.getDate());
```

Comment:
Is it not enough with 'setFullYear'? Is that just not setting the same date you are getting or am I missing something?